src/test/java/com/epam/tc/hw3/page/objects/fluents/JdiIndexFluentPage.java

```diff
@@ -0,0 +1,102 @@
+package com.epam.tc.hw3.page.objects.fluents;
+
+import static com.epam.tc.hw3.AbstractBaseTest.TIMEOUT_SECONDS;
+
+import com.epam.tc.hw3.page.objects.composite.components.BottomPartComponent;
+import com.epam.tc.hw3.page.objects.composite.components.HeaderComponent;
+import com.epam.tc.hw3.page.objects.composite.components.LeftSectionComponent;
+import com.epam.tc.hw3.page.objects.composite.components.ProfileComponent;
+import java.time.Duration;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class JdiIndexFluentPage {
+
+    private final WebDriver driver;
+    private final WebDriverWait wait;
+    private final ProfileComponent profile;
+    private final HeaderComponent header;
+    private final BottomPartComponent bottomPart;
+    private final LeftSectionComponent leftSection;
+    private final String indexPageUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    @FindBy(tagName = "iframe")
+    private List<WebElement> frames;
+
+    @FindBy(xpath = "//*[@class='uui-navigation nav navbar-nav m-l8']"
+        + "//*[contains(text(),'Service')]")
+    private WebElement serviceButton;
```

Comment:
Isn't it enough to use just //*[contains(text(),'Service')] as locator?

Reply:
2 different web elements have 'Service' as their text, so it's necessary to have more precise locators